misc/config_tools/xforms/misc_cfg.h.xsl

```diff
@@ -97,7 +98,12 @@
       </xsl:otherwise>
     </xsl:choose>
   </xsl:variable>
-  <xsl:value-of select="acrn:define('SOS_BOOTARGS_DIFF', concat($quot, $bootargs, ' ', $maxcpus, ' ', $quot), '')" />
+  <xsl:variable name="hugepage">
```

Comment:
Shouldn't the name of this variable be `hugepages` (with an 's' at the end)?

Reply:
Config tool will add the hugepage variable that declares the string "hugepagesz=1G hugepages=[size]" into sos kernel cmdline. The hugepages variable only mean the value of hugepages in the string "hugepagesz=1G hugepages=[size]".

Reply:
I see, thanks for the explanation. Should we use a different name, less close to the `hugepages` variable name? Perhaps something like `hugepage_kernelstring`?